input_raw.go

```diff
@@ -78,7 +78,7 @@ func (i *RAWInput) listen(address string) {
 	host, port, err := net.SplitHostPort(address)
 
 	if err != nil {
-		log.Fatal("input-raw: error while parsing address", err)
+		log.Fatalf("input-raw: error while parsing address: %s , err: %v" , address, err)
```

Comment:
err already has address. I think it should smth like 
`log.Fatalf("input-raw: error while parsing address: %s", err)`